src/ussd_popi_rapidpro.js

```diff
@@ -927,37 +927,194 @@ go.app = function() {
             });
         });
 
-        self.add("state_opt_out", function(name) {
+        self.add("state_optout_menu", function(name) {
```

Comment:
I don't think we need to rename this state, the mainmenu still references "state_opt_out"

Reply:
oh wait, I see what's happening, I think we just need to change the reference in "state_main_menu" to point here